examples/sample.js

```diff
@@ -20,7 +20,7 @@ export default {
   },
   margin: { x: '2.5cm', top: '2cm', bottom: '1.5cm' },
   content: [
-    { text: 'Lorem ipsum', bold: true, fontSize: 24, margin: { bottom: 10 } },
+    { text: 'Lorem ipsum', bold: true, fontSize: 24, margin: { bottom: 10 }, textAlign: 'center'},
```

Comment:
Apparently, this file is not linted. But there is a space missing before the `}`.